packages/neuron-ui/src/tests/validators/outputs/fixtures.ts

```diff
@@ -21,7 +21,7 @@ const fixtures = {
       ],
       ignoreLastAmount: false,
     },
-    exception: true,
+    exception: false,
```

Comment:
The title is `should throw an error when address is not provided` doesn't match `exception: false`

Maybe a specific case of `empty address with checked "send max"` should be added.

Reply:
Added a test case to check that `ignoreLastAddress` is true.

Reply:
The condition becomes complex with adding a new field `ignoreLastAddress` because the combination of `ignoreLastAmount && ignoreLastAddress` includes 4 cases but only `false && false`, `false && true`, `true && false` are covered.

What if using one flag to skip validation of the last fieldset

Reply:
That reminds me! I added `ignoreLastAddress` to ignore the last items when checking whether the last max button is disabled.
How about removing the last item to check the max button's status? Then the `validateOutputs` is unnecessary to change. https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/components/Send/index.tsx#L137

Reply:
I didn't get the meaning of `removing the last item to check the max button's status`

Reply:
https://github.com/nervosnetwork/neuron/pull/2932/files#diff-40f3dad0e46ccc5c6d81f579bd16c048e233df26fc686e22a864b55f75da8522R151-R156
Remove the last item of output when checking the max button's status.

Reply:
> [#2932 (files)](https://github.com/nervosnetwork/neuron/pull/2932/files#diff-40f3dad0e46ccc5c6d81f579bd16c048e233df26fc686e22a864b55f75da8522R151-R156) Remove the last item of output when checking the max button's status.

LGTM

Reply:
The logic that `outputs.slice(0, -1)` can be reused where `ignoreLastAmount = true` so the logic of `validateOutputs` can be simplified. We can update it later.